src/player.cpp

```diff
@@ -1212,6 +1215,11 @@ R"(EasyRPG Player - An open source interpreter for RPG Maker 2000/2003 games.
       --start-map-id N     Overwrite the map used for new games and use.
                            MapN.lmu instead (N is padded to four digits).
                            Incompatible with --load-game-id.
+      --autobattle-algo A  Which AutoBattle algorithm to use.
+                           Possible options:
+                            RPG_RT     - The default RPG_RT compatible algo, including RPG_RT bugs
+                            RPG_RT+    - The default RPG_RT compatible algo, with bug fixes
+                            ATTACK     - RPG_RT+ but only physical attacks, no skills
```

Comment:
The lines are too long (>80) but this is a problem in many parts of the help output

Reply:
Would like to punt until we have a wholistic answer to this one